docs/conf.py

```diff
@@ -101,6 +102,10 @@ def get_version():
     'confval': 'tooltip',  # for custom object
 }
 
+rst_epilog = """
```

Comment:
This is the section causing issues with docs-lint. I'm not sure what a great solution is yet.

Reply:
I got it fixed up by ignoring them for now.